xml/gl.xml

```diff
@@ -42242,7 +42242,7 @@ typedef unsigned int GLhandleARB;
                 <!-- <command name="glTexturePageCommitmentEXT"/> -->
             </require>
         </extension>
-        <extension name="GL_EXT_sparse_texture2" supported="gl"/>
+        <extension name="GL_EXT_sparse_texture2" supported="gl|gles2"/>
```

Comment:
I didn't have any rigorous process for this. I would kinda glance at the extension spec and go off what it said about required GL versions, or otherwise guess. The only thing it drives in this repo is what goes in glcorearb.h, which I suspect is barely used by anyone, but happy to be more accurate.